modules/DMZ/data_kit/data_splitting.py

```diff
@@ -1,10 +1,18 @@
-from modules.toolbox import *
 import pandas as pd
 from sklearn import preprocessing
 from sklearn import cross_validation
-
-
-# Helper functions for the whole framework to use
+import numpy as np
+
+
+# Gets the training and testing splits for training.
```

Comment:
@asclines actually, where would you suggest having the documentation? I was thinking the folder layer above could include an overview of what the data_kit contains